content/docs/reference/config/annotations/index.html

```diff
@@ -0,0 +1,290 @@
+
+<table class="message-fields">
+<thead>
+<tr>
+<th>Annotation</th>
+<th>Type</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr id="sidecar.istio.io/inject">
+<td><code>sidecar.istio.io/inject</code></td>
+<td><code>boolean</code></td>
+<td>
+<p>The sidecar injector will inject the sidecar into pods by default. Add the sidecar.istio.io/inject annotation with value false to the pod template spec to override the default and disable injection.</p>
+
+</td>
+</tr>
+<tr id="sidecar.istio.io/status">
+<td><code>sidecar.istio.io/status</code></td>
+<td><code>string</td>
+<td>
+<p>TODO</p>
+
+</td>
+</tr>
+<tr id="sidecar.istio.io/proxyImage">
+<td><code>sidecar.istio.io/proxyImage</code></td>
+<td><code>string</td>
+<td>
+<p>TODO Docker image</p>
+
+</td>
+</tr>
+<tr id="sidecar.istio.io/interceptionMode">
+<td><code>sidecar.istio.io/interceptionMode</code></td>
+<td><code>string</td>
+<td>
+<p>See <a href="/docs/setup/kubernetes/istio-cni-install/#traffic-redirection-details">Traffic Redirection</a>.</p>
+
+</td>
+</tr>
+<tr id="status.sidecar.istio.io/port">
+<td><code>status.sidecar.istio.io/port</code></td>
+<td><code>string</td>
+<td>
+<p>See <a href="/docs/setup/kubernetes/istio-cni-install/#traffic-redirection-details">Traffic Redirection</a>.</p>
+
+</td>
+</tr>
+<tr id="readiness.status.sidecar.istio.io/initialDelaySeconds">
+<td><code>readiness.status.sidecar.istio.io/initialDelaySeconds</code></td>
+<td><code>unsigned</td>
+<td>
+<p>TODO</p>
+
+</td>
+</tr>
+<tr id="readiness.status.sidecar.istio.io/periodSecond">
+<td><code>readiness.status.sidecar.istio.io/periodSecond</code></td>
+<td><code>unsigned</td>
+<td>
+<p>TODO</p>
+
+</td>
+</tr>
+<tr id="readiness.status.sidecar.istio.io/failureThreshold">
+<td><code>readiness.status.sidecar.istio.io/failureThreshold</code></td>
+<td><code>unsigned</td>
+<td>
+<p>TODO</p>
+
+</td>
+</tr>
+<tr id="readiness.status.sidecar.istio.io/applicationPorts">
+<td><code>readiness.status.sidecar.istio.io/applicationPorts</code></td>
+<td><code>string</td>
+<td>
+<p>TODO comma separated list or *</p>
+
+</td>
+</tr>
+<tr id="traffic.sidecar.istio.io/includeOutboundIPRanges">
+<td><code>traffic.sidecar.istio.io/includeOutboundIPRanges</code></td>
+<td><code>string</td>
+<td>
+<p>TODO comma separated CIDR list</p>
+
+</td>
+</tr>
+<tr id="traffic.sidecar.istio.io/excludeOutboundIPRanges">
+<td><code>traffic.sidecar.istio.io/excludeOutboundIPRanges</code></td>
+<td><code>string</td>
+<td>
+<p>TODO comma separated CIDR list</p>
+
+</td>
+</tr>
+<tr id="traffic.sidecar.istio.io/includeInboundPorts">
+<td><code>traffic.sidecar.istio.io/includeInboundPorts</code></td>
+<td><code>string</td>
+<td>
+<p>TODO</p>
+
+</td>
+</tr>
+<tr id="traffic.sidecar.istio.io/excludeInboundPorts">
+<td><code>traffic.sidecar.istio.io/excludeInboundPorts</code></td>
+<td><code>string</td>
+<td>
+<p>TODO comma separated list or *</p>
+
+</td>
+</tr>
+<tr id="sidecar.istio.io/v1alpha1/statsInclusionPrefixes">
```

Comment:
Several corrections here
1. change annotation name
`sidecar.istio.io/statsInclusionPrefixes`
2. Instead of `Example` add the default values for inclusion which are `listener_manager,cluster_manager,cluster.xds-grpc,server,http_mixer_filter,tcp_mixer_filter`
3. State that this replaces the default list above.  These following stats:`listener_manager,cluster_manager` , are required for proper functioning of the readiness subsystem therefore you should include these two with your list.

@geeknoid what do you think is it better to just fix this in code, or doc is fine?

Reply:
Is there a PR for `sidecar.istio.io/statsInclusionPrefixes` in the code itself or should I also do that work separately now?

When I implemented statsInclusionPrefixes I did not have this guidance.  The current implementation works as it says.  I am unclear about alpha/beta versioning of annotations.  Should we use `alpha.sidecar.istio.io/statsInclusionPrefixes` to indicate it is an experimental feature?